src/drt/README.md

```diff
@@ -49,34 +49,36 @@ detailed_route
     [-repair_pdn_vias layer]
 ```
 
-- `-output_maze` output maze log filename
-- `-output_drc` output drc report filename
-- `-output_cmap` output congestion map file file
-- `-output_guide_coverage` output guide coverage filename
-- `-drc_report_iter_step` report drc on each iteration which is a multiple of this step
-- `-db_process_node` specify the process node (optional)
-- `-disable_via_gen` option to diable via generation with bottom and top routing layer
-- `-droute_end_iter` number of detailed routing iterations
-- `-via_in_pin_bottom_layer` via-in pin bottom layer name  
-- `-via_in_pin_top_layer` via-in pin top layer name
-- `-or_seed` random seed for the order of nets to reroute. 
-- `-or_k` number of swaps is given by $k * sizeof(rerouteNets)$
-- `-bottom_routing_layer` bottommost routing layer name 
-- `-top_routing_layer` topmost routing layer name
-- `-verbose` set verbose if value is greater than 0. 
-- `-param` you may set parameters in this argument using a file. 
-**note that `-param` may not be used with other keyword arguments.
-(deprecated)**
-- `-distributed` enable distributed mode with Kubernetes and Google Cloud, [guide](./doc/Distributed.md).
-- `-remote_host` the host IP 
-- `-remote_port` the value of the port to access from 
-- `-shared_volume` the mount path of the nfs shared folder
-- `-cloud_size` the number of workers 
-- `-clean_patches` clean unneeded patches during detailed routing. 
-- `-no_pin_access` disables pin access for routing
-- `-min_access_points` minimum access points for standard cell and macro cell pins. 
-- `-save_guide_updates` save guides updates 
-- `-repair_pdn_vias` this option is used for some PDKs which have M1 and M2 layers that run in parallel. 
+#### Options
+
+| Switch Name | Description |
+| ----- | ----- |
+| `-output_maze` | output maze log filename |
+| `-output_drc` | output drc report filename |
+| `-output_cmap` | output congestion map file |
+| `-output_guide_coverage` | output guide coverage filename |
+| `-drc_report_iter_step` | report drc on each iteration which is a multiple of this step |
+| `-db_process_node` | specify the process node (optional) |
+| `-disable_via_gen` | option to diable via generation with bottom and top routing layer |
+| `-droute_end_iter` | number of detailed routing iterations |
+| `-via_in_pin_bottom_layer` | via-in pin bottom layer name |
+| `-via_in_pin_top_layer` | via-in pin top layer name |
+| `-or_seed` | random seed for the order of nets to reroute | 
+| `-or_k` | number of swaps is given by $k * sizeof(rerouteNets)$ |
+| `-bottom_routing_layer` | bottommost routing layer name |
+| `-top_routing_layer` | topmost routing layer name |
+| `-verbose` | set verbose if value is greater than 0. |
+| `-param` | you may set parameters in this argument using a file.  **note that `-param` may not be used with other keyword arguments. (deprecated)** |
```

Comment:
@maliberty I think the param file has been deprecated for 2+ years now. Time to remove it?

Reply:
#3646